### PR TITLE
GLX: fixups

### PR DESCRIPTION
**Pull request by fritsch**
Fixes: http://sprunge.us/VIZA - root cause the visuals did not match.

### CODE REVIEW

**Comment by fritsch** [comment]
Removed two commits. The above code is enought to make it work again.

**Comment by fritsch** [comment]
jenkins build this please

File: xbmc/windowing/X11/GLContextGLX.cpp [hunk 1]
```diff
@@ -153,10 +153,6 @@ bool CGLContextGLX::IsSuitableVisual(XVisualInfo *vInfo)
     return false;
   if (glXGetConfig(m_dpy, vInfo, GLX_BLUE_SIZE, &value) || value < 8)
     return false;
-  if (glXGetConfig(m_dpy, vInfo, GLX_ALPHA_SIZE, &value) || value < 8)
-    return false;
-  if (glXGetConfig(m_dpy, vInfo, GLX_DEPTH_SIZE, &value) || value < 8)
```
Comment (FernetMenta) [inline xbmc/windowing/X11/GLContextGLX.cpp]: why do you drop depth check?

Comment (fritsch) [reply xbmc/windowing/X11/GLContextGLX.cpp]: Was done by mistake. 8 is enough? For EGL we set it to 16 on certain platforms and to 24 on others. What do we really need?

Comment (FernetMenta) [reply xbmc/windowing/X11/GLContextGLX.cpp]: it should be 24, aligned with this: https://github.com/xbmc/xbmc/blob/master/xbmc/windowing/X11/GLContextEGL.cpp#L284

Comment (fritsch) [reply xbmc/windowing/X11/GLContextGLX.cpp]: Thx. made the check for 24

**Pull request merged**